test/test_torch.py

```diff
@@ -12975,8 +13006,22 @@ def check_sum_all(tensor):
         else:
             check_sum_all(torch.tensor([True, False, True], dtype=torch.bool, device=device))
 
-    def _test_memory_format_transformations(self, device, input_generator_fn, transformation_fn,
-                                            compare_data=True, default_is_preserve=False):
+    @onlyCPU
+    def test_sum_bfloat16(self, device):
```

Comment:
Please don't create separate tests when actually a datatype should be added to existing sum test

Reply:
This case is trying to compare the accuracy between BFloat16 and Float32 and the `sum` test cases consist of  `test_sum_all`, `test_sum_out`, `test_sum_noncontig` and other cases.